test/integration_test.go

```diff
@@ -214,21 +218,114 @@ func Test_RpaasApi(t *testing.T) {
 		err = api.unbind(instanceName, helloServiceHost)
 		require.NoError(t, err)
 
+		time.Sleep(10 * time.Second)
```

Comment:
Are these sleeps necessary? Maybe we could wait for the nginx service endpoints to become available?